### ISSUE TITLE
Consider adding copy of *.rar files in DockerFile

### ISSUE THREAD

**Issue by sorenviggo** [larshp/nwabap751#7]
Please consider a prerequisite step as suggested in https://github.com/nzamani/sap-nw-abap-trial-docker
The step would be to create a local folder called 'sapdownloads', and then adding all .rar installation files to this.
The DockerFile then should have this line added.
`COPY sapdownloads /tmp`

Then no other manual steps would be needed.

**Comment by larshp** [comment]
yeah, I havent really maintained this for a while, the one by @nzamani is good :)

**Comment by sorenviggo** [comment]
I have tested both, but actually find yours to be superior, due to the local '/tmp/install.sh' script you are creating, to perform the unrar locally in the container.
But with very little effort, you could adopt 2 small enhancements from @nzamani, then yours would be perfect.
If you add the prerequisite step with creating the folder, and adding below 2 lines in the DockerFile, then you can install the entire SAP system, by running only 3 commands in the cmd. 
```
RUN mkdir /run/uuidd && chown uuidd /var/run/uuidd && /usr/sbin/uuidd
COPY sapdownloads /tmp
```
Brgds, Søren.

**Comment by larshp** [comment]
Pull requests welcome 😄
